src/components/AvatarList/AvatarList.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { AvatarList } from './AvatarList';
+
+export const mockAvatars = [
+	{
+		initials: 'BN',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'KN',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'BC',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'AC',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'AC',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'AC',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'AC',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+];
+
+storiesOf('AvatarList', module)
+	.addParameters({ jest: ['AvatarList', 'AvatarIcon'] })
+	.add('AvatarList (closed dropdown)', () => <AvatarList avatars={mockAvatars} isOpen={false} />)
```

Comment:
To test the tooltip open/closed behavior we add a story component who manages the isOpen state.
You can look at the dropdown component for inspiration:
https://github.com/viaacode/avo2-components/blob/release/v1.5.0/src/components/Dropdown/Dropdown.stories.tsx#L9

Reply:
Tooltip gaat in een ander PR komen te staan, dit is gwn hetgeen @benjaminnaesen al voorzien had.

Reply:
ah, ok, i see